src/bioregistry/data/bioregistry.json

```diff
@@ -2885,18 +2890,18 @@
     "miriam": {
       "deprecated": false,
       "description": "An Archival Resource Key (ARK) is a Uniform Resource Locator (URL) that is a multi-purpose persistent identifier for information objects of any type.",
-      "homepage": "http://n2t.net/",
+      "homepage": "https://arks.org/",
```

Comment:
rather than overriding the external data inside the `miriam` dictionary, we can make a top-level "homepage" key and put this there.

Reply:
See 64978d9a